src/dlgSourceEditorArea.cpp

```diff
@@ -20,14 +21,37 @@
 
 
 #include "dlgSourceEditorArea.h"
+#include "edbee/edbee.h"
+#include "edbee/texteditorwidget.h"
+#include "edbee/models/textdocument.h"
+#include "edbee/views/textrenderer.h"
+#include "edbee/views/texttheme.h"
+#include "edbee/models/textgrammar.h"
+#include "edbee/models/texteditorconfig.h"
 
-#include "THighlighter.h"
-
+//#include "ui_source_editor_area.h"
 
 dlgSourceEditorArea::dlgSourceEditorArea(QWidget* pF) : QWidget(pF)
 {
     // init generated dialog
     setupUi(this);
-    highlighter = new THighlighter(editor->document());
-    editor->setTabStopWidth(25);
+
+    // Configuring the editor widget with defaults
+
+    edbee::TextEditorConfig* config = edbeeEditorWidget->config();
+
+    config->beginChanges();
+
+    config->setSmartTab(true); // I'm not fully sure what this does, but it says "Smart" so it must be good
+    config->setCaretBlinkRate(200);
+
+    config->setIndentSize(2); // 2 spaces is the Lua default
```

Comment:
Yes it is, see any code written in Programming in Lua, the Lua Reference Manual, and Beginning Lua Programming.